seesaw/indices/roibased/preprocessor.py

```diff
@@ -151,14 +158,17 @@ def preprocess_roi_dataset(
     ims = []
     paths = []
     #excluded = []
-    start = 60000
```

Comment:
this start variable looks suspect. are you committing changes that were not meant to be committed?